src/test/play-3/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/welshlanguagetoggle/exampleWelsh.scala.html

```diff
@@ -1,11 +1,16 @@
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 @import uk.gov.hmrc.hmrcfrontend.views.html.components._
+@import play.api.Configuration
 
-@this(hmrcLanguageSelect : HmrcLanguageSelect)
+@this(
+        govukServiceNavigation: GovukServiceNavigation,
+        hmrcServiceNavigationLanguageSelect: HmrcServiceNavigationLanguageSelect
+)
 
-@()
+@()(implicit messages: Messages, configuration: Configuration)
```

Comment:
seems good to me, seeing it again like this, I do wish we didn't have to pass implicit Configuration :oof: but then it should only be happening in 1 or 2 templates and layout at that and not likely have to be passed down through lots of views so 👍

Reply:
Yes, we basically been an instance of Configuration to be able to retrieve this value: "language.fallback.url" which we hardcode in the reference.conf to be: "https://www.gov.uk/government/organisations/hm-revenue-customs". I wonder if there's a better way of doing this before we release this change?